src/main/java/com/tasker/server/boards/rest/dao/BoardDaoImpl.java

```diff
@@ -19,4 +25,35 @@ public void create(Board board) {
         jdbcTemplate.update("INSERT INTO tasker.boards(id, title, created_at, description) VALUES (?, ?, ?, ?)",
             board.getId(), board.getTitle(), board.getCreatedAt(), board.getDescription());
     }
+
+    @Transactional
+    @Override
+    public List<Board> fetchBoards() {
+
+        String sql = "SELECT id, title, description, created_at FROM tasker.boards";
+        return  jdbcTemplate.query(sql, new BoardRowMapper());
+    }
+
+    @Override
+    public Board fetchBoard(UUID boardId) {
+
+        String sql = "SELECT id, title, description, created_at FROM tasker.boards WHERE id = ?";
+        return jdbcTemplate.queryForObject(sql, new Object[] {boardId}, new BoardRowMapper());
+    }
+
+    private static final class BoardRowMapper implements RowMapper<Board> {
+
```

Comment:
needless space

Reply:
should we use some kind of linter? do you know one for java?

Reply:
I don't. Probably there are

Reply:
ok , I'll create issue for that.